### ISSUE TITLE
Spy-Report list is broken with latest game update

### ISSUE THREAD

**Issue by leobut** [ogame-infinity/web-extension#308]
URL: https://s256-de.ogame.gameforge.com/game/index.php?page=messages

Console shows:

```
Uncaught TypeError: Cannot read properties of null (reading 'getAttribute')
    at ogkush.js:1:385781
    at NodeList.forEach (<anonymous>)
    at OGInfinity.spyTable (ogkush.js:1:383566)
    at ogkush.js:1:429023
```

**Comment by Bishop341-B** [comment]
Fixed with #305

**Comment by leobut** [comment]
Amazing, thanks for the quick reply and action - much appreciated 👍
